pkg/backend/display/progress.go

```diff
@@ -619,8 +622,17 @@ func (display *ProgressDisplay) printDiagnostics() bool {
 				display.println("")
 			}
 		}
+	}
 
+	// Print a link to Copilot to explain the failure.
+	// Check for SuppressPermalink ensures we don't print the link for DIY backends
+	if wroteDiagnosticHeader && !display.opts.SuppressPermalink && !display.opts.SuppressLinkToCopilot {
+		display.println("    " +
+			colors.SpecCreateReplacement + "[Pulumi Copilot]" + colors.Reset + " Would you like help with these diagnostics?")
+		display.println("    " +
+			colors.Underline + colors.Blue + display.permalink + "?explainFailure" + colors.Reset)
```

Comment:
It's not obvious to me why this is better than `?askCopilot`. Even if the query parameter was only a key, no value, I think that's clearer to me than `?explainFailure`.

Reply:
> It's not obvious to me why this is better than `?askCopilot`. Even if the query parameter was only a key, no value, I think that's clearer to me than `?explainFailure`.

This is based on feedback from the team - url-encoded textual query looks 'ugly' in the CLI and requires a new release of the CLI (plus user upgrading their CLI) when we have to change it.

Reply:
I'm suggesting we have it be `?askCopilot` with no URl encoded query. Just `?askCopilot`.